tests/compose/test.py

```diff
@@ -22,7 +22,7 @@
 def stop(exit_code):
     print_logs()
     sys.stdout.flush()
-    print(subprocess.check_output("docker-compose -f " + compose_file + " down", shell=True).decode())
+    print(subprocess.check_output("docker compose -f " + compose_file + " down", shell=True).decode())
```

Comment:
I think we should start using f-strings here. 

```suggestion
    print(subprocess.check_output(f"docker compose -f {compose_file} down", shell=True).decode())
```